### PR TITLE
WIP: Initial air drop

### PR DESCRIPTION
**Pull request by mxmauro**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,9 @@
 *.~*
+package-lock.json
```
Comment (jijordre) [inline .gitignore]: package-lock.json is needed for production deployment in order to assure consistent third party package versions.

Comment (mxmauro) [reply .gitignore]: Yes, although it is recommended to include it, I'm in the 50% that thinks this json is a weird patch that gives more problems and benefits.

Comment (jijordre) [reply .gitignore]: I can see your point in that it is not a flawless solution. Yet for production deployments we want to make certain that the third-party dependencies are predictable.

Thus as long as we are aware of the possible caveats involved using this file e.g. for local development I'd like to keep this file outside of the _.ignore_. For local development it is just a matter of deleting this file for the corner cases where it makes problems.

File: package-lock.json [hunk 2]
```diff
@@ -1,8012 +0,0 @@
-{
```
Comment (jijordre) [inline package-lock.json]: Arguments given for the _.gitignore_ also suggest that this file be kept under version control.

**Pull request closed without merging**